bin/dump-dev-program-config

```diff
@@ -7,17 +7,18 @@ source bin/lib.sh
 docker::set_network_name_dev
 
 docker run --rm -it \
+  -v "$(pwd):/project" \
   -e "PGPASSWORD=example" \
   --network "${DOCKER_NETWORK_NAME}" \
   postgres:12.5 \
-  /usr/bin/pg_dump \
-  -w \
-  -Fc \
-  -h db \
-  -U postgres \
-  -t programs \
-  -t questions \
-  -t versions \
-  -t versions_programs \
-  -t versions_questions \
-  postgres > dev_programs.dump
+  /bin/bash -c "pg_dump \
+    -w \
+    -Fc \
+    -h db \
+    -U postgres \
+    -t programs \
+    -t questions \
+    -t versions \
+    -t versions_programs \
+    -t versions_questions \
+    postgres > /project/dev_db.dump"
```

Comment:
Nit: I think you need to update this name in the docs at the top of this file

Reply:
aha, reverted to the original name. thanks!